src/config.ts

```diff
@@ -238,6 +238,19 @@ export const MEMPOOL_POLLING_INTERVAL_MS = +env.varOrDefault(
   '30000', // 30 seconds
 );
 
+//
+// AWS settings
+//
+//
+export const AWS_ACCESS_KEY_ID = env.varOrUndefined('AWS_ACCESS_KEY_ID');
```

Comment:
@djwhitt should we add these variables to docs or wait for the featured to be tested before we "expose" it?

Reply:
I'd say go ahead and add them. That way we can direct any community members who want to test this to the docs.

Reply:
[`60e3b62` (#139)](https://github.com/ar-io/ar-io-node/pull/139/commits/60e3b626971f301ff800050cacca78bb62c8c506)